src/locales/default.json

```diff
@@ -355,12 +355,14 @@
       "information": "The minimum amount of voting power required to create a proposal"
     },
     "allowOnlyAuthors": "Allow only authors to submit a proposal",
+    "enableAliased": "Allow voting with session keys",
     "editValidation": "Edit validation",
     "selectValidation": "Select validation",
     "validationParameters": "Validation parameters",
     "voting": "Voting",
     "votingDelay": "Voting delay",
     "votingPeriod": "Voting period",
+    "votingAliased": "Setting affects voting, creating and deleting proposals",
```

Comment:
```suggestion
    "votingAliased": "Voting, creating and deleting proposals will no longer require the user to manually sign each time.",
```

Reply:
@bonustrack @metrox-eth do you think we should go into more details about how it works or how the UX will be in this tooltip?